api/proto/config/vm.proto

```diff
@@ -41,4 +41,9 @@ message VmConfig {
   uint32 vncDisplay = 17;
   string vncPasswd = 18;
   bool disableLogs = 19;
+  // Set if the VM runs in a mode with pinned CPUs.  In this case all the VCPUs
```

Comment:
Nit but "runs" sounds like status and "should run" would read more like configuration.

Reply:
Will fix!